src/data/people.json

```diff
@@ -333,6 +333,15 @@
         "Reut Tsarfaty"
       ]
     },
+    {
+      "name": "Refael Shaked Greenfeld",
+      "startYear": 2020,
+      "advisors": [
+        "Reut Tsarfaty"
+      ],
+      "homepage": "https://www.linkedin.com/in/refael-shaked-greenfeld/",
+      "interests": "Deep Learning & NLP, Scuba Diving"
+    }
```

Comment:
there's a missing comma here, for this to be a valid JSON

Reply:
Oops. thanks for the fix!